### ISSUE TITLE
Add a LICENSE

### ISSUE THREAD

**Issue by alanorth** [nairobilug/nairobilug.or.ke#123]
So we have no license in this repository. The repository contents are purely authored but LUG people, so we can choose one ourselves.

Since it's not really code we should probably use some form of Creative Commons, no? Perhaps [Creative Commons 4.0 BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)? That requires attribution and sharing under the same terms, and allows for commercial use (as long as the result is attributed to us and shared under CC-BY-SA as well. This is similar to the GPL for code, which is not against commercial use).

We would also need an item in `README.md` listing the license terms.

**Comment by alanorth** [comment]
This issue probably needs a vote, as it covers content written by many authors.

**Comment by raymondwanyoike** [comment]
I'd vote for a CC license, but I'm not an author. Should we invite every post author to this issue?

**Comment by alanorth** [comment]
Yeah, we should do that actually, so we can put our hand over our heart and say it's "this license", and advertise it in the blog footer, README.md, etc.

**Comment by BonfaceKilz** [comment]
Long overdue(!) but I've seen this discussion on some Guile mailing list. CC BY-SA seems like a good fit, since it's compatible with GPLv3 and Wikipedia at the same time. The draw back with CC BY-SA is that it's not FDL-compatible. Perhaps a freer CC0 license would suffice?

**Comment by bkmgit** [comment]
Perhaps put all rights reserved on each post where there is no explicit license.  As each post has an author or authors, this can be updated as needed.
cc @PaulMayero @Lochipi @faithuniterh @afterthesegfault
